api/tests/routes/native/v1/account_test.py

```diff
@@ -1543,12 +1543,11 @@ def test_validate_phone_number_with_first_sent_code(self, client, app):
         first_token = create_phone_validation_token(user, first_number)
         create_phone_validation_token(user, second_number)
 
-        response = client.post("/native/v1/validate_phone_number", {"code": first_token.value})
+        response = client.post("/native/v1/validate_phone_number", {"code": first_token.encoded_token})
 
-        assert response.status_code == 204
+        assert response.status_code == 400
         user = users_models.User.query.get(user.id)
-        assert user.is_phone_validated
-        assert user.phoneNumber == first_number
+        assert not user.is_phone_validated
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Dans ce cas, il faut aussi renommer le test